cmd/platform_observer/main.go

```diff
@@ -32,19 +32,23 @@ func main() {
 		logger.Fatal("No APIs to observe")
 	}
 
-	minInterval := viper.GetDuration("observer.min_poll")
 	backlogTime := viper.GetDuration("observer.backlog")
+	minInterval := viper.GetDuration("observer.min_poll")
+	maxInterval := viper.GetDuration("observer.max_poll")
+	if minInterval >= maxInterval {
+		logger.Fatal("minimum block polling interval cannot be greater or equal than maximum")
```

Comment:
why log this?

Reply:
If the min is greater the min, we going to trigger a fatal error to stop the application, because can bug the logic

Reply:
Maybe you can write code so you can’t break logic? 

There is a reason why min, max function exist, to avoid such code